core/commonMain/src/kotlinx/serialization/Serializers.kt

```diff
@@ -137,7 +136,7 @@ internal fun <T : Any> SerializersModule.reflectiveOrContextual(kClass: KClass<T
  *
  * The recommended way to retrieve the serializer is inline [serializer] function and [`serializer(KType)`][serializer]
  *
- * This API is not guaranteed to work consistent across different platforms or
+ * This API is not guaranteed to work consistently across different platforms or
```

Comment:
pls fix it in the docs for `KClass.serializer()` too, then

Reply:
Already done :)